### DISCUSSION TITLE
Hide a discussion

### DISCUSSION THREAD

**Discussion by trevthom** [stackernews/stacker.news#694] [Ideas]
I would love the ability to hide a discussion that doesn't interest me at all. Sometimes I will click a discussion that I want to look at later and decide if I want to zap it, but sorting through other discussions makes it difficult to find (and I don't want to bookmark it).

**Comment by ekzyis** [comment]
Closing this because of [this comment](https://github.com/stackernews/stacker.news/issues/695#issue-2045209586):

> (Just realized the 'Issues' section has the ability for feature request so ignore the one in 'Discussions')
